feat(config): added yale fingerprint series

catduckgnaf opened the pull request:
Feat(config): Added YRD410-F-ZW3, YRD420-F-ZW3, YRD430-F-ZW3 & YRD450-F-ZW3

YRD410-F-ZW3 Assure 2 Lock Touch SL Keyed Pushbutton Deadbolt ID 0x4936
YRD420-F-ZW3 Assure 2 Lock Touch SL Keyed Touchscreen Deadbolt ID(s) 0x4929
YRD430-F-ZW3 Assure 2 Lock Touch SL Key Free Pushbutton Deadbolt  ID 0x4934
YRD450-F-ZW3 Assure 2 Lock Touch SL Key Free Touchscreen Deadbolt  ID(s) 0x4913, 0x4928

catduckgnaf committed:
Added Fingerprint Version for Yale Locks

With Fingerprint

apella12 commented:
No comment as to content, but I believe your title should be `feat(config): added ...` if you want to clear the error.

catduckgnaf commented:
> No comment as to content, but I believe your title should be `feat(config): added ...` if you want to clear the error.

thanks, think i got it now

catduckgnaf commented:
Did I get the title right or do I need to do anything else?

apella12 commented:
I'm not the reviewer/approver.  I just thought clearing the error would get more attention from the reviewer.  My observation is he/they are pretty busy.  Have you added these files to the store/config folder and reinterview ?  I think will get them working locally (but not entirely sure).

catduckgnaf commented:
You are right. I know they are. I also have discovered a missing device id. Im gonna close and look. It works locally.

catduckgnaf committed:
Merge branch 'zwave-js:master' into master

catduckgnaf committed:
Yale Fingerprint Updates

Updated Yale Fingerprint for YRD410-F, 420-F, 430-F, 450-F.
Also updated yale_template.json

catduckgnaf committed:
mining naming corrections

catduckgnaf committed:
minor edits

catduckgnaf committed:
Update yale_template.json

catduckgnaf committed:
removing extra _

catduckgnaf committed:
Update index.ts

catduckgnaf committed:
Update yrd420-f-zw3.json

apella12 commented:
Again a tip, but no review. Take a quick look at the details of the failing tests for clues.  There is an extra `],` on line 11 of one file.  There may be more, but that's one.  I don't know how you code, but I use VSC and there is some checking built into the node-js repository.  (still make mistakes though..)

drizuid commented on packages/config/config/devices/0x0129/yrd420-f-zw3.json:
```diff
@@ -0,0 +1,62 @@
+	"manufacturerId": "0x0129",
+	"label": "YRD420-F-ZW3",
+	"description": "Assure 2 Lock Touch SL Key Free Touchscreen Deadbolt",
+	"devices": 
```
missing `[`
`"devices":` should be `"devices": [`
otherwise the closing bracket for the array on line 11 will break the code.

catduckgnaf reviewed (commented):
Thank you, looks good. I appreciate you correcting that.

AlCalzone commented:
@catduckgnaf do you have a manual for the locks you added?
#6972 was just merged and contains some of the locks with product type `0x8107`.

You have some others in here with different product type/id, but apparently the same name. Would be good to verify what's the difference there. There are usually manuals for the product family explaining the different product types and IDs:
*	"productType": "0x8104",
	"productId": "0x4936"

*	"productType": "0x8107",
	"productId": "0x4929"

*	"productType": "0x8107",
	"productId": "0x4934"

*	"productType": "0x8107",
	"productId": "0x4913"

*	"productType": "0x8107",
	"productId": "0x4928"

oandrew commented:
@AlCalzone 
This one covers it: 
https://products.z-wavealliance.org/ProductManual/File?folder=&filename=cert_portal/certs/785/YaleAssureZWavePlusSystemIntegratorsGuidev2.3_YRC_D_200_400_600_ZW3.pdf
From https://github.com/zwave-js/node-zwave-js/discussions/5391
